src/metabase/util/date_2.clj

```diff
@@ -210,7 +210,11 @@
   (keyword ((requiring-resolve 'metabase.public-settings/start-of-week))))
 
 (def ^:private ^{:arglists '(^java.time.DayOfWeek [k])} day-of-week*
-  (u.date.common/static-instances DayOfWeek))
+  (let [m (u.date.common/static-instances DayOfWeek)]
```

Comment:
unrelated change?

Reply:
I put this in there to catch errors when I had something blowing up because it was calling `day-of-week*` with `:::::sunday` -- I thought better error validation never hurts